source/namespace_map.yml

```diff
@@ -0,0 +1,6 @@
+spec_format: HDMF
+data_type_inc: data_type_inc
+data_type_def: data_type_def
+data_types: data_types
+api: hdmf
+version: '2.0.0b'
```

Comment:
I agree that the ``version``, and anything else that we can pull in from the JSON schema file is preferable to avoid having to specify the same information in multiple places. It may also make sense to put this file at the root of the repo to have the setup files in one place.

Reply:
`make html` etc. works with hdmf-schema-language as the cwd, though it feels like it shouldn't...